apps/condo/domains/meter/tasks/sendVerificationDateReminder.js

```diff
@@ -179,30 +179,46 @@ const getOrganizationLang = async (context, id) => {
     return get(COUNTRIES, get(organization, 'country.locale'), DEFAULT_LOCALE)
 }
 
-const generateReminderMessages = async ({ context, reminderWindowSize, reminders }) => {
+const generateReminderMessages = async ({ context, reminders }) => {
     const messages = []
     await Promise.all(reminders.map(async (reminder) => {
         const { meter, residents } = reminder
         const lang = await getOrganizationLang(context, meter.organization.id)
+
+        // get context for specific locale
+        const localeDependedContext = {
+            ...context.createContext({ skipAccessControl: true }),
+            req: { locale: lang },
+        }
+
+        // get list of meter resources for certain locale
+        const localizedResources = await MeterResource.getAll(localeDependedContext, {})
+
         // prepare a message for each resident
         messages.push(
-            ...residents.map(resident => ({
-                sender: { dv: 1, fingerprint: 'meters-validation-date-cron-push' },
-                to: { user: { id: resident.user.id } },
-                type: METER_VERIFICATION_DATE_REMINDER_TYPE,
-                lang,
-                meta: {
-                    dv: 1,
-                    data: {
-                        reminderDate: dayjs(meter.nextVerificationDate).locale(lang).format('D MMM'),
-                        meterId: meter.id,
-                        resource: { name: get(meter, 'resource.name', '') },
-                        userId: resident.user.id,
-                        residentId: resident.id,
-                        url: `${conf.SERVER_URL}/meter`,
+            ...residents.map(resident => {
```

Comment:
no need for `...`, map creates new array

Reply:
In case if I remove that destructurization as a result I'll get [[{}]] structure (arrays in aray), but I actually expect array of objects instead